### PR TITLE
Revert "Remove superfluous variable." for issue #1779

### PR DESCRIPTION
**Pull request by hgtonight**
This reverts commit b06639891474779f16d5c3df6c17815a8f5bcc2d.

This reverts a bad commit for the sake of brevity. Yay testing!

### CODE REVIEW

**Comment by hgtonight** [comment]
This really closes #1779 on the 2.1 branch

**Comment by linc** [comment]
Huh?

**Comment by hgtonight** [comment]
In my previous PR to close #1179, @kaecyra suggested I remove the `$Locale = Gdn::Locale();` line as it seems superfluous. I made the suggested change, tested it as working and committed. It was merged ~4 months later.

While submitting the same patch against master, I noticed that I used the `$Locale` variable twice and introduced a regression into 2.1. Sure enough, when I had tested my patch previously, I was testing the original code. This PR fixes that mistake.

**tl;dr I smatched it.**
